main.py

```diff
@@ -0,0 +1,63 @@
+# Imports
+from flask import Flask, render_template
+import requests, json
+import pandas as pd
+import datetime
+import mysql.connector
+from sqlalchemy import create_engine
+import threading
+from data import select_user
+
+app = Flask(__name__)
+
+def main():
+	
+	# Calls the main() function every 300 seconds 
+	threading.Timer(300.0, main).start()
```

Comment:
You should use crontab instead

Reply:
I'm sorry. I tried to look up about running a python script using crontab, but couldn't figure it out.

Reply:
Try using this - https://gitlab.com/doctormo/python-crontab/